preparations.html

```diff
@@ -54,18 +53,10 @@ <h3>Section Two - DSiWare Game</h3>
                     “Guitar Rock Tour”</li>
                     <li>
                         “The Legend of Zelda: Four Swords”</li></ul></li>
-                <li>You need to legitimately own one of the listed games on the device you want to install CFW on. <strong>Pirated copies won't work.</strong><br/>
-                </li>
-                <li>Open the eShop (update your device if necessary)</li>
-                <li>Open the Menu</li>
-                <li>Go to Settings and more</li>
-                <li>Go to Scan QR Code</li>
-                <li>Scan any QR Code from <a href="https://3ds.hacks.guide/installing-boot9strap-(dsiware-game-injection-list)">the website linked above</a>, making sure it is a game form your region.
-                <br/>
-                I recommend SUDOKU by EA, as it is just 2$ and the game this exploit was originally discovered in</li>
-                <li>Buy the game</li>
-                <li>Download the game</li>
-
+                <li>Press L+R in the home menu to activate camera</li>

```

Comment:
Readd notice of needing a legitimate copy

Reply:
A person with a stock 3ds shouldn't have the ability to use pirated copies in the first place. This warning was for the people doing system transfer with an hacked 3ds, in which the tickets for the pirated copy will not transfer.

Reply:
True!